library/src/main/java/com/h6ah4i/android/widget/advrecyclerview/animator/impl/BaseItemAnimationManager.java

```diff
@@ -158,7 +157,16 @@ protected void endAnimation(RecyclerView.ViewHolder holder) {
     }
 
     protected void resetAnimation(RecyclerView.ViewHolder holder) {
-        AnimatorCompatHelper.clearInterpolator(holder.itemView);
```

Comment:
Is checking holder for null not necessary?

Reply:
Shouldn't be necessary, unless we've seen NPE come from this method before, given the code has always directly accessed holder